lambdas/death-enrichment/src/test/resources/uk/gov/di/data/lep/approvals/DeathEnrichmentTest.approvalTestForOutput.approved.txt

```diff
@@ -0,0 +1 @@
+{"aud":null,"events":{"https://ssf.account.gov.uk/v1/deathRegistration":{"deathDate":{"description":null,"value":"2007-03-06"},"deathRegistrationID":1234567890,"freeFormatDeathDate":null,"deathRegistrationTime":{"value":"2023-03-06T09:30:50.000"},"subject":{"address":[{"addressCountry":null,"addressLocality":null,"buildingName":null,"buildingNumber":null,"departmentName":null,"dependentAddressLocality":null,"dependentStreetName":null,"doubleDependentAddressLocality":null,"organisationName":null,"postalCode":"GT8 5HG","streetName":null,"subBuildingName":null,"uprn":null,"validFrom":null,"validUntil":null}],"birthDate":[{"description":null,"value":"1967-03-06"}],"name":[{"description":null,"nameParts":[{"type":"GivenName","value":"ERICA","validFrom":null,"validUntil":null},{"type":"FamilyName","value":"BLOGG","validFrom":null,"validUntil":null}],"validFrom":null,"validUntil":null},{"description":"Name before marriage","nameParts":[{"type":"GivenName","value":"ERICA","validFrom":null,"validUntil":null},{"type":"FamilyName","value":"Death notification name","validFrom":null,"validUntil":null}],"validFrom":null,"validUntil":null}],"sex":["Female"]}}},"exp":null,"iat":1,"iss":null,"jti":"guid_1","nbf":null,"sub":null,"toe":1678095050,"txn":"guid_2"}
```

Comment:
This is deliberately not a json approval or file, because we care that the raw format of the message hasn't changed rather than anything else